test/integration/Bid.test.ts

```diff
@@ -38,39 +60,51 @@ describe('Bid', () => {
 
         testDataService = app.IoC.getNamed<TestDataService>(Types.Service, Targets.Service.TestDataService);
         bidService = app.IoC.getNamed<BidService>(Types.Service, Targets.Service.BidService);
+        marketService = app.IoC.getNamed<MarketService>(Types.Service, Targets.Service.MarketService);
+        bidDataService = app.IoC.getNamed<BidDataService>(Types.Service, Targets.Service.BidDataService);
+        listingItemService = app.IoC.getNamed<ListingItemService>(Types.Service, Targets.Service.ListingItemService);
 
         // clean up the db, first removes all data and then seeds the db with default data
-        await testDataService.clean([], false);
+        await testDataService.clean([]);
+
+        let defaultMarket = await marketService.getDefault();
+        defaultMarket = defaultMarket.toJSON();
+        log.debug('defaultMarket: ', defaultMarket);
+
+        createdListingItem = await testDataService.create<ListingItem>({
+            model: 'listingitem',
+            data: {
+                market_id: defaultMarket.id,
+                hash: 'itemhash'
+            } as any,
+            withRelated: true
+        } as TestDataCreateRequest);
     });
 
     afterAll(async () => {
         //
     });
 
-
     test('Should throw ValidationException because there is no listing_item_id', async () => {
         expect.assertions(1);
-        await bidService.create(testData).catch(e =>
+        await bidService.create(testData as BidCreateRequest).catch(e =>
             expect(e).toEqual(new ValidationException('Request body is not valid', []))
         );
     });
 
-
     test('Should create a new bid', async () => {
-        testData['listing_item_id'] = 1;
-        const bidModel: Bid = await bidService.create(testData);
-        createdId = bidModel.Id;
-
+        testData['listing_item_id'] = createdListingItem.id;
```

Comment:
add listing_item_id to BidCreateRequest.
cast the testdata object as BidCreateRequest so you can do: testData.listing_item_id.